### ISSUE TITLE
Don't focus Scatter Desktop after logging in to app

### ISSUE THREAD

**Issue by Soleone** [GetScatter/ScatterDesktop#20]
### Problem

After logging into a website like https://scattellet.com or https://dexeos.io that app didn't end up being focussed but instead Scatter Desktop was showing me the Identity page.

### Expected

I would expect a more seamless process where the app/website retains focus and I don't have to worry about the Scatter main window at all most of the time.

### Replication steps

1. Visit e.g. https://dexeos.io
2. Click Sign in through scatter
3. Select scatter identity in scatter popup
4. Select eos account in same scatter popup
5. End up in Scatter instead of back at the dexeos.io

![image](https://user-images.githubusercontent.com/2923/44170749-4a844d80-a0a6-11e8-8bbe-db59740b2c8d.png)

![image](https://user-images.githubusercontent.com/2923/44170778-638cfe80-a0a6-11e8-9a0c-8eec96c63e8d.png)

**Comment by nsjames** [comment]
So I looked into this and this is actually a pretty painful problem. The app would need to be minimized after the popup is closed to make it so that the main window doesn't pop up after. A user-fix for this is simply minimizing Scatter to the tray which negates the main window popping up. However, when done through code this would minimize Scatter's main window to the tray as well, and clicking on the app on the dock ( macos ) would not open it.

Gonna have to keep looking into this and seeing how we can try to focus the last window ( not sure if this is even possible ) that the user had focused.

**Comment by Soleone** [comment]
For what it's worth this is now fixed on OSX when using the new Minimize to Tray button 👍

**Comment by nsjames** [comment]
Right, that definitely works. The only downside with that is that it's a user-action fix. I'm still looking for a real fix for this though so I'll leave this ticket open.

**Comment by jjssoftware** [comment]
I tested this on Linux Mint Cinnamon with scatter desktop 9.3.0 in both scattllet and deseos and it looks fine now as far as I can see. 

I tested with both the scatter window open behind the browser window and also minimized to the tray. In both cases the scatter logon prompt window popped into the foreground, I was able to login and then following that the browser window had focus with no sign of scatter. Looks quite seamless to me. 

I don't know if this issue is O/S specific and the problem does still exist elsewhere.
